tests/test_routes.py

```diff
@@ -182,35 +183,21 @@ def test_create_record_invalid_content_type(self):
 
     def test_delete_inventory_record_success(self):
         """Test to check if it deletes an inventory record"""
-        test_inventory_record = self._create_inventory_records(1)[0]
-
-        response = self.client.get(f"{BASE_URL}/{test_inventory_record.product_id}", json=test_inventory_record.serialize())
-        # check if record was created successfully
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-        request_body = {
-            "product_id": test_inventory_record.product_id,
-            "condition": test_inventory_record.condition.value
-        }
-
-        response = self.client.delete(f"{BASE_URL}/{test_inventory_record.product_id}", json=request_body)
+        test_record = self._create_inventory_records(1)[0]
+        response = self.client.delete(f"{BASE_URL}/{test_record.product_id}/{test_record.condition.name}")
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.assertEqual(len(response.data), 0)
-        
         # make sure they are deleted
-        response = self.client.get(f"{BASE_URL}/{test_inventory_record.product_id}", json=request_body)
+        response = self.client.get(f"{BASE_URL}/{test_record.product_id}", json=test_record.serialize())
```

Comment:
we will need to pass condition also to this since the read function will also be updated

Reply:
the read hasn't been updated yet, whoever refactores read, will have to change it or individually we can change later. as of now read works with `product_id` so i dont think it would make sense to ship code with `condition`.

Reply:
yeah to be remembered during backlog